Sources/Holocron/UserDefaults+StorageProvider.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  UserDefaults+StorageProvider.swift
+//  Holocron-iOS
+//
+//  Created by Pranjal Satija on 4/10/19.
+//  Copyright © 2019 CocoaPods. All rights reserved.
+//
+
+import Foundation
+
+public extension StorageProvider {
+    typealias UserDefaults = Foundation.UserDefaults
+}
+
+extension UserDefaults: StorageProvider {
```

Comment:
Should be good to make this extension public here too

Reply:
Same as above. Can't make an extension `public` if it declares conformance.